app/controllers/articles_controller.rb

```diff
@@ -1,14 +1,17 @@
 class ArticlesController < ApplicationController
+
+  before_action :dummyLogForBeforeAction
+
   def index
     articles=Article.joins(:author).select('articles.*, authors.name AS author_name') #implementd joins
-    render json: articles, status: 200
+    render json: articles.as_json, status: 200
   end
 
   def show
     begin
       article=Article.find(params[:id])
       if article
-        render json: article, status: 200
+        render json: article.as_json, status: 200
```

Comment:
send only limited data, instaed of full article data

Reply:
Changes implemented, please check latest commit.